packages/calling-stateful-client/src/InternalCallContext.ts

```diff
@@ -142,23 +139,44 @@ export class InternalCallContext {
 
   public setLocalRenderInfo(
     callId: string,
+    streamKey: string,
```

Comment:
is stream key the type of stream? `Key: 'screenshare' | 'video'` or something?

Reply:
updated